Wrappers/Python/cil/optimisation/algorithms/SIRT.py

```diff
@@ -50,19 +50,24 @@ class SIRT(Algorithm):
     upper : :obj:`float`, default = None
             Upper bound constraint, default value = :code:`-inf`.
     constraint : IndicatorBox function, default = None
-                 Enforce box constraint using the :class:`.IndicatorBox` function.
+                A constraint, e.g., :class:`.IndicatorBox` function is enforced in every iteration.
     kwargs:
         Keyword arguments used from the base class :class:`.Algorithm`.    
 
     Note 
     ----
     
+    If :code:`constraint` is not passed, then :code:`lower` and :code:`upper` are looked at.
+
     If :code:`constraint` is passed, it should be an :class:`.IndicatorBox` function, 
```

Comment:
Say explicitly that if constraint is passed, then it takes precedence over lower and upper, which are then ignored.

Reply:
fixed